model_analyzer/result/result_manager.py

```diff
@@ -194,17 +194,22 @@ def create_tables(self,
             # Measurements
             results = self._state_manager.get_state_variable(
                 "ResultManager.results")
-            for measurement in results.get_list_of_measurements():
-                for gpu_uuid, gpu_metrics in measurement.gpu_data().items():
+
+            for run_config_measurement in results.get_list_of_run_config_measurements(
+            ):
+                for gpu_uuid, gpu_metrics in run_config_measurement.gpu_data(
+                ).items():
                     for gpu_metric in gpu_metrics:
                         if gpu_metric.tag not in gpu_specific_metrics_from_measurements:
                             gpu_specific_metrics_from_measurements[
-                                gpu_metric.tag] = gpu_metric
+                                gpu_metric[0]] = gpu_metric
```

Comment:
what is [0] in this case?

Reply:
This should be tag - I changed this when debugging. Reverted this change.